### ISSUE TITLE
Error when installing from source

### ISSUE THREAD

**Issue by DiegoGM91** [qiboteam/qibojit#61]
Cloning the repo and running

`pip install -r requirements.txt`

according to the documentation, gives the following error:

`ERROR: Could not open requirements file: [Errno 2] No existe el archivo o el directorio: 'requirements.txt'`

**Comment by scarrazza** [comment]
I think you are looking at the stable documentation (which refers to an old tag of qibojit). If you select latest this command is not there anymore.

**Comment by DiegoGM91** [comment]
Right. Thanks @scarrazza 

Still, when trying to install it from source I had to manually run `pip install packaging` before running `pip install .` I don't know if this is to be expected.

**Comment by scarrazza** [comment]
In principle it is not, e.g. github actions do not need anything in order to install from source.

**Comment by scarrazza** [comment]
@DiegoGM91 could you please list all commands you perform in order to raise this issue?

**Comment by DiegoGM91** [comment]
Hi @scarrazza 

I just run `pip install .` in an environment where `qibo 0.1.6` was installed too, if I remember correctly.

However, I've tried to replicate this issue in a new clean environment, without success.

The installation now automatically installs `packaging` (which I had to install manually last time I raised the issue), even when I previously uninstalled it from the environment.

So I couldn't tell what happened (maybe something related to `pip` or `setuptools` or even `packaging` version in the old environment perhaps?)
